src/kayenta/reducers/selectedConfig.ts

```diff
@@ -111,9 +111,9 @@ const metricList = handleActions(
     [Actions.ADD_METRIC]: (state: ICanaryMetricConfig[], action: Action & any) =>
       idMetrics(state.concat([action.payload.metric])),
     [Actions.REMOVE_METRIC]: (state: ICanaryMetricConfig[], action: Action & any) =>
-      idMetrics(state.filter(metric => metric.id !== action.payload.id)),
+      idMetrics(state.filter((metric) => metric.id !== action.payload.id)),
```

Comment:
Did prettier change its default wrapping of single values? Seems like it! That's fine! I'll submit a followup PR to re-prettier everything.

Reply:
[I think so!](https://prettier.io/docs/en/options.html#arrow-function-parentheses)